repos/spack_repo/builtin/packages/glx/package.py

```diff
@@ -14,9 +14,18 @@ class Glx(BundlePackage):
 
     version("1.4")
 
+    # GLX is only supported on Linux-like platforms
+    conflicts("platform=windows")
+    conflicts("platform=darwin")
+
     depends_on("libglx")
     provides("gl@4.5")
 
+    def setup_dependent_build_environment(
+        self, env: EnvironmentModifications, dependent_spec: Spec
+    ):
+        env.prepend_path("OpenGL_ROOT", self.home)
```

Comment:
Shouldn't this go into `cmake_prefix_paths`?

```suggestion
    @property
    def cmake_prefix_paths(self):
        return self.home
```

(also, just curious, but what's `home` and how does it differ from `prefix`?)

Reply:
From what I could tell it was already going into that path. The issue is, the prefix path is only used for searching for cmake config/CPS. I had to explicitly set `OpenGL_ROOT` here so that when that module calls `find_library` it would search that path first. See #2057 and [cmake#27299](https://gitlab.kitware.com/cmake/cmake/-/issues/27299)